From 6cb0c196f1a8baff86c75d0d11b04198e83b65ad Mon Sep 17 00:00:00 2001
From: Leonidas Spyropoulos <artafinde@archlinux.org>
Date: Thu, 13 Apr 2023 10:29:13 +0100
Subject: [PATCH] feat: remove variable fields from manifest

Including the gradle,jdk versions along with kernel versiona dn JDK
might be variable in different systems and affects the hash of each jar.

Signed-off-by: Leonidas Spyropoulos <artafinde@archlinux.org>
---
 build.gradle | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/build.gradle b/build.gradle
index 10c8dda..0003ea5 100644
--- a/build.gradle
+++ b/build.gradle
@@ -53,11 +53,7 @@ jar {
     }
     manifest {
         attributes(
-            'Built-By'       : System.properties['user.name'],
             'Build-Revision' : versioning.info.commit,
-            'Created-By'     : "Gradle ${gradle.gradleVersion}",
-            'Build-Jdk'      : "${System.properties['java.version']} (${System.properties['java.vendor']} ${System.properties['java.vm.version']})",
-            'Build-OS'       : "${System.properties['os.name']} ${System.properties['os.arch']} ${System.properties['os.version']}",
             'Version'        : "${gradle.rootProject.version}"
         )
     }
